hogvm/__tests__/date.hog

```diff
@@ -0,0 +1,29 @@
+let dt := fromUnixTimestamp(1234334543)
+print(dt)
+print(toString(dt))
+print(toInt(toUnixTimestamp(dt)))
```

Comment:
Commenting more generally - having all these functions as globals - is this something we will regret later?
Things like `toDate` or `now` are pretty generic variable names (not mentioning a lot of the other ones we have). Do we need to consider namespacing things already?

Reply:
The problem is with any function -> you can define your own `print`, `fetch`, etc functions. I'm not being creative with the names, but just copying from ClickHouse. Since currently we don't support lambdas (we only have global functions), this won't bite anyone... but yeah. I guess people will be able to override global function names with local variables.

If we're on the subject, I'd like to one day make `var.asdf(arg1)` an alternate way to write `asdf(var, arg1)` for any `var`, `asdf`, etc. It's like this in nim and I find it really handy, though we might need stricter types for this.